prisma-fmt/tests/text_document_completion/scenarios/datasource_default_completions/result.json

```
@@ -0,0 +1,55 @@
+{
+  "isIncomplete": false,
+  "items": [
+    {
+      "label": "engines provider",
+      "kind": 5,
+      "documentation": {
+        "kind": "markdown",
+        "value": "```prisma\nprovider = \"foo\"\n```\n___\nDescribes which datasource connector to use. Can be one of the following datasource providers: `postgresql`, `mysql`, `sqlserver`, `sqlite`, `mongodb` or `cockroachdb`.\n\n"
+      },
+      "insertText": "provider = $0",
+      "insertTextFormat": 2
+    },
+    {
+      "label": "engines url",
+      "kind": 5,
+      "documentation": {
+        "kind": "markdown",
+        "value": "```prisma\nurl = \"String\" | env(\"ENVIRONMENT_VARIABLE\")\n```\n___\nConnection URL including authentication info. Each datasource provider documents the URL syntax. Most providers use the syntax provided by the database. [Learn more](https://pris.ly/d/connection-strings).\n\n"
+      },
+      "insertText": "url = $0",
+      "insertTextFormat": 2
+    },
+    {
+      "label": "engines shadowDatabaseUrl",
+      "kind": 5,
+      "documentation": {
+        "kind": "markdown",
+        "value": "```prisma\nshadowDatabaseUrl = \"String\" | env(\"ENVIRONMENT_VARIABLE\")\n```\n___\nConnection URL including authentication info to use for Migrate's [shadow database](https://pris.ly/d/migrate-shadow).\n\n"
+      },
+      "insertText": "shadowDatabaseUrl = $0",
+      "insertTextFormat": 2
+    },
+    {
+      "label": "engines directUrl",
+      "kind": 5,
+      "documentation": {
+        "kind": "markdown",
+        "value": "```prisma\ndirectUrl = \"String\" | env(\"ENVIRONMENT_VARIABLE\")\n```\n___\nConnection URL for direct connection to the database. [Learn more](https://pris.ly/d/data-proxy-cli).\n\n"
+      },
+      "insertText": "directUrl = $0",
+      "insertTextFormat": 2
+    },
+    {
+      "label": "engines relationMode",
+      "kind": 5,
+      "documentation": {
+        "kind": "markdown",
+        "value": "```prisma\nrelationMode = \"foreignKeys\" | \"prisma\"\n```\n___\nSet the global relation mode for all relations. Values can be either \"foreignKeys\" (Default), or \"prisma\". [Learn more](https://pris.ly/d/relation-mode)\n\n"
+      },
+      "insertText": "relationmode = $0",
```

If we decide the completion to be a bit different (see above comment), then maybe:

```suggestion
      "insertText": "relationmode = \"$0\"",
```